### PR TITLE
Feature/core 422 add v select

### PR DESCRIPTION
**Pull request by DanielMaass**

### CODE REVIEW

File: dapps/components.vue/src/components/forms/forms.vue [hunk 1]
```diff
@@ -86,7 +86,7 @@
           id="evan-form-test-4"
           label="Field 4 - Select"
           type="number"
-          v-model="field4"
+          v-model="select"
```
Comment (Tschuck) [inline dapps/components.vue/src/components/forms/forms.vue]: The select parameter isn't defined within the `forms.ts`. Is it correct?

File: dapps/components.vue/src/components/forms/forms.ts [hunk 2]
```diff
@@ -57,6 +57,7 @@ export default class Forms extends mixins(EvanComponent) {
   field2 = 'a';
   field3 = 0;
   field4 = 0;
+  field5 = null;
```
Comment (Tschuck) [inline dapps/components.vue/src/components/forms/forms.ts:60]: Can we also add a `v-select` sample to the vue form? Could be something like this:

```
  vSelect: {
    value: '',
    validate: function(vueInstance: Forms, form: SampleFormInterface) {
      return this.value.length !== 0;
    },
    uiSpecs: {
      type: 'v-select',
      attr: {
       options: this.contacts,
       taggable:true
      }
    }
  },
```

**Review by Tschuck — CHANGES_REQUESTED** [review]

File: dapps/components.vue/src/components/forms/forms.vue (hunk 1 above)
Comment (DanielMaass) [reply dapps/components.vue/src/components/forms/forms.vue]: Right! I fixed that.

File: dapps/components.vue/src/components/forms/forms.ts (hunk 2 above)
Comment (DanielMaass) [reply dapps/components.vue/src/components/forms/forms.ts:60]: solved :)

**Pull request merged**